spec/features/admin_claim_tasks_update_with_dqt_api_spec.rb

```diff
@@ -639,7 +639,7 @@ def task_outcome
           let(:data) do
             {
               qts_award_date: Date.new(
-                claim.eligibility.first_eligible_itt_academic_year.start_year - 1.year,
```

Comment:
Quickly looking at the code I think the fix for this might be to remove `.start_year`.